docs/lint-checks.rst.j2

```diff
@@ -0,0 +1,35 @@
+{#
+  This template is rendered by conf.py & Sphinx when building documentation.
+
+  The reST comment below should land in the rendered file, to discourage people
+  from editing it instead of this template.
+#}
+
+..
+   Please, do not edit this file, is is rendered from lint-checks.rst.j2,
+   and all your changes would be overwritten.
+
+:orphan:
+
+.. _lint-checks:
+
+Lint checks
+===========
+
+{% macro emit_table(title, linters) %}
+.. list-table:: {{ title }}
```

Comment:
A minor nitpick: Using headings instead of table headers has the advantage that the section appears in the left menu and the section id (html link anchor) is not numbered.

Reply:
Fixed in 8d07fe55.